kvdb-rocksdb/CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ The format is based on [Keep a Changelog].
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
+### Changed
+- Default column support removed from the API
+  - Column argument type changed from `Option<u32>` to `u32`
+  - Migration `None` -> `0`, `Some(0)` -> `1`, `Some(1)` -> `2`, etc.
+- `DatabaseConfig::default()` defaults to 1 column
+- `Database::with_columns` still accepts `u32`, but panics if `0` is provided 
```

Comment:
let's also mention that the db should be opened with +1 number of columns for migration